js/app/Modal.html

```diff
@@ -321,6 +321,8 @@
     const items = JSON.parse(app.pages.form.elements.form.items.value);
     const item = items.find(i => i.description == itemDescription);
     if (! item || item.serialized) {
+      // TODO: Fix bug that causes this error to be shown when clicking on a student.
```

Comment:
Is this `TODO` done or does it still need work?

Reply:
regarding this, I can fix it, but I think the recent changes that make that field not selectable and the cursor not change to a pointer make it clear that this is not a valid selection